build.sbt

```diff
@@ -10,11 +10,11 @@ normalizedName := "laminarexamples"
 
 organization := "com.raquo"
 
-scalaVersion := "2.13.1"
+scalaVersion := "2.13.4"
 
-crossScalaVersions := Seq("2.12.10", "2.13.1")
+crossScalaVersions := Seq("2.12.10", "2.13.1", "2.13.4")
```

Comment:
You should remove 2.13.1 if you're adding 2.13.4

Reply:
Done. Thanks ... new to cross building :-/